src/unix/freebsd.c

```diff
@@ -269,30 +269,6 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
 }
 
 
-int uv__sendmmsg(int fd, struct uv__mmsghdr* mmsg, unsigned int vlen) {
-#if __FreeBSD__ >= 11 && !defined(__DragonFly__)
```

Comment:
Do we need to keep the `#else` for `__DragonFly__`?

Refs: https://github.com/libuv/libuv/pull/3040#issuecomment-722980369
cc @devnexen

Reply:
Right... so I guess I should just drop the `defined(__DragonFly__)` everywhere.